x/evm/keeper/keeper.go

```diff
@@ -164,9 +164,9 @@ func (k Keeper) GetAccountStorage(ctx sdk.Context, address common.Address) (type
 
 // GetChainConfig gets block height from block consensus hash
 func (k Keeper) GetChainConfig(ctx sdk.Context) (types.ChainConfig, bool) {
-	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefixChainConfig)
+	store := ctx.KVStore(k.storeKey)
 	// get from an empty key that's already prefixed by KeyPrefixChainConfig
```

Comment:
are we still getting an empty key prefix? i think this might need updating

Reply:
see above

Reply:
this is in regards to the comment, should update because we are no longer getting an empty key prefix but we are using one that is set in types.KeyPrefixChainConfig